samples/cpp/videocapture_depth.cpp

```diff
@@ -0,0 +1,174 @@
+#include "opencv2/videoio.hpp"
+#include "opencv2/highgui.hpp"
+#include "opencv2/imgproc.hpp"
+
+#include <iostream>
+
+using namespace cv;
+
+static void colorizeDisparity(const Mat& gray, Mat& rgb, double maxDisp=-1.f)
+{
+    CV_Assert(!gray.empty());
+    CV_Assert(gray.type() == CV_8UC1);
+
+    if(maxDisp <= 0)
+    {
+        maxDisp = 0;
+        minMaxLoc(gray, 0, &maxDisp);
+    }
+
+    rgb.create(gray.size(), CV_8UC3);
+    rgb = Scalar::all(0);
+    if(maxDisp < 1)
+        return;
+
+    Mat tmp;
+    convertScaleAbs(gray, tmp, 255.f / maxDisp);
+    applyColorMap(tmp, rgb, COLORMAP_JET);
+}
+
+static float getMaxDisparity(VideoCapture& capture)
+{
+    const int minDistance = 400; // mm
```

Comment:
It depends on camera optics and may be different. I propose to add it to command line parameters with default value and promote as function parameter here.